tedge/src/cli/connect/command.rs

```diff
@@ -338,6 +338,21 @@ fn new_bridge(
         }
     }
 
+    match cloud {
+        Cloud::C8y => {
+            println!("Checking if tedge-agent and sm-c8y mapper are installed.\n");
+            if which("tedge_agent").is_err() && which("tedge_mapper").is_err() {
+                println!("Warning: tedge_mapper is not installed. We recommend to install it.\n");
```

Comment:
Wouldn`t that also say tedge_mapper isn't installed even if it is installed but not tedge_agent?

Keep in mind that having thin-edge without software management is an absolutely valid installation. Points out of that:
1. Please remove the "We recommend...."
2. Shouldn't we only check for tedge_agent here?
3. For the updated message please use: "Software management is not installed. Skipping enabling related components."

Reply:
Without tedge_mapper and just with tedge_agent the software management is not complete right?